server/services/interventionsService.ts

```diff
@@ -624,7 +624,7 @@ export default class InterventionsService {
     token: string,
     referralId: string,
     reasonCode: string,
-    cancellationComments: string
+    cancellationComments: string | null
```

Comment:
Are we happy sending `null` values, or do we want it to be `undefined`?

Reply:
sending null values is kind of an antipattern in our API since they are ignored, but it's fine. 

the reason is that you may reasoably expect a null value to update the value to null. i.e. in a PATCH endpoint where something is already set, you might expect sending null to 'unset' it. but we don't support that. and generally discourage sended the value at all.